packages/artifact/README.md

```diff
@@ -0,0 +1,199 @@
+# `@actions/artifact`
+
+## Usage
+
+You can use this package to interact with the actions artifacts.
+- [Upload an Artifact](#Upload-an-Artifact)
+- [Download a Single Artifact](#Download-a-Single-Artifact)
+- [Download All Artifacts](#Download-all-Artifacts)
+- [Additional Documentation](#Additional-Documentation)
+
+Relative paths and absolute paths are both allowed. Relative paths are rooted against the current working directory.
+
+## Upload an Artifact
+
+Method Name: `uploadArtifact`
+
+#### Inputs
+ - `name`
+    - The name of the artifact that is being uploaded
+    - Required
+ - `files`
```

Comment:
In the new version of the artifact action we discussed supporting wild cards.  Is the intent here that an action that uses this library can support wildcards but it must pass fully qualified paths to the module?

Reply:
Originally I had the wildcard search directly backed into this package using `@actions/glob`, however, we decided to move it out (that will go into `@actions/upload-artifact` which will then call this package with the search result). This gives users more flexibility in terms of how they use this package, they can even use something else for search if `@actions/glob` doesn't fit their needs. This package just solely focuses on interactions with artifacts at a low level. 

Both relative and absolute paths work (everything gets normalized and resolved behind the scenes), so fully qualified paths are not required